toolkit/tools/pkg/imagecustomizerlib/typeConversion.go

```diff
@@ -81,7 +87,16 @@ func partitionToImager(partition imagecustomizerapi.Partition, fileSystems []ima
 		return configuration.Partition{}, fmt.Errorf("failed to find mount point with ID (%s)", partition.Id)
 	}
 
-	imagerEnd, _ := partition.GetEnd()
+	imagerStart := partition.Start / diskutils.MiB
+	if partition.Start%diskutils.MiB != 0 {
```

Comment:
nit: spaces around `%`?

Reply:
Alas, Go's formatter prefers it this way. :-(